pipeline/tests/pil.rs

```diff
@@ -173,13 +173,6 @@ fn test_external_witgen_fails_on_conflicting_external_witness() {
     verify_test_file(f, Default::default(), external_witness).unwrap();
 }
 
-#[test]
-fn test_global() {
-    verify_pil("pil/global.pil", Default::default());
-    // Halo2 would take too long for this.
-    // Starky requires at least one witness column, this test has none.
-}
-
```

Comment:
why was this removed?

Reply:
Because the new optimizer removes everything in that file causing an error with pil-verify. The file also does not contain anything meaningful.